pkg/client/application_profile.go

```diff
@@ -31,6 +31,30 @@ func (h *V1Client) GetApplicationProfileByName(profileName string) (*models.V1Ap
 
 	return nil, fmt.Errorf("Application profile '%s' not found.", profileName)
 }
+func (h *V1Client) GetApplicationProfileByNameAndVersion(profileName string, version string) (*models.V1AppProfileSummary, string, string, error) {
```

Comment:
Let's leave only this function and remove GetApplicationProfileByName()

Reply:
removed GetApplicationProfileByName() and changed logic accordingly